v4/export/sql.go

```diff
@@ -294,19 +294,23 @@ func buildOrderByClause(conf *Config, db *sql.Conn, database, table string) (str
 		}
 		if ok {
 			return "ORDER BY _tidb_rowid", nil
-		} else {
-			return "", nil
 		}
 	}
-	pkName, err := GetPrimaryKeyName(db, database, table)
+	cols, err := GetPrimaryKeyColumns(db, database, table)
 	if err != nil {
 		return "", withStack(err)
 	}
-	tableContainsPriKey := pkName != ""
-	if tableContainsPriKey {
-		return fmt.Sprintf("ORDER BY `%s`", escapeString(pkName)), nil
-	}
-	return "", nil
+        if len(cols) == 0 {
+	        return "", nil
+        }
+        odSql := "ORDER BY "
+        for i, col := range cols {
+                odSql += fmt.Sprintf("`%s`", escapeString(col));
+                if i != len(cols) - 1 {
+                        odSql += ", "
+                }
+        }
```

Comment:
These lines' indentation looks strange. Can you keep it the same as the previous lines'?

Reply:
Ok, wait a moment.